packages/hebao_v1/contracts/stores/QuotaStore.sol

```diff
@@ -85,9 +87,7 @@ contract QuotaStore is DataStore
         public
         onlyWalletModule(wallet)
     {
-        Quota storage q = quotas[wallet];
-        q.spentAmount = spentQuota(wallet).add(amount).toUint128();
-        q.spentTimestamp = uint64(block.timestamp);
+        _addToSpent(wallet, quotas[wallet], amount);
```

Comment:
shall we change `function addToSpent`'s visibility to `external`?

Reply:
I think making at least all functions that are `onlyWalletModule` `external` makes a lot of sense. I think in some select cases `public` may be marginally more efficient than `external` but it's so small it doesn't really matter.